core/style.css

```
@@ -1223,24 +1223,46 @@ table.artifact .portal {
 .cellscore .ui-accordion-header-active:before {
 	content: "⊟";
 }
+.cellscore table {
+	width: 90%;
+	max-width: 360px; /* prevent width change on scrollbar appearance (after animation) */
+}
+
+/* prevent nonfunctional horizontal scrollbar in Chrome (perhaps jQuery issue) */
+.cellscore .historychart > div {
+  overflow: auto; 
+}
+
+.cellscore .logscale {
+  vertical-align: middle;
+  margin-top: 0;
+}
 
 @-moz-document url-prefix() {
  .cellscore .logscale {
-    transform: scale(0.5) translate(0,-18px);
-    vertical-align: top !important;
+    transform: scale(0.8);
+  }
+
+  /* prevent nonfunctional vertical scrollbar in Firefox (perhaps jQuery issue) */
+  .cellscore > .historychart {
+    overflow-y: hidden;
```

jQuery sets `overflow: auto;` inline, so we must use `!important` to redefine the style.

```suggestion
    overflow-y: hidden !important;
```